internal/app/oauth/client.go

```diff
@@ -0,0 +1,98 @@
+package oauth
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/caos/oidc/pkg/oidc"
+	"github.com/caos/oidc/pkg/op"
+)
+
+type ConfClient struct {
+	ID                 string
+	applicationType    op.ApplicationType
+	authMethod         oidc.AuthMethod
+	accessTokenType    op.AccessTokenType
+	responseTypes      []oidc.ResponseType
+	grantTypes         []oidc.GrantType
+	allowedScopes      []string
+	redirectURIs       []string
+	logoutRedirectURIs []string
+	loginURI           string
+	iDTokenLifetime    time.Duration
+	clockSkew          time.Duration
+	devMode            bool
+}
+
+func (c *ConfClient) GetID() string {
+	return c.ID
+}
+
+func (c *ConfClient) RedirectURIs() []string {
+	return c.redirectURIs
+}
+
+func (c *ConfClient) PostLogoutRedirectURIs() []string {
+	return c.logoutRedirectURIs
+}
+
+func (c *ConfClient) LoginURL(id string) string {
+	return fmt.Sprintf(c.loginURI, id)
+}
+
+func (c *ConfClient) ApplicationType() op.ApplicationType {
+	return c.applicationType
+}
+
+func (c *ConfClient) AuthMethod() oidc.AuthMethod {
+	return c.authMethod
+}
+
+func (c *ConfClient) IDTokenLifetime() time.Duration {
+	return c.iDTokenLifetime
+}
+
+func (c *ConfClient) AccessTokenType() op.AccessTokenType {
+	return c.accessTokenType
+}
+
+func (c *ConfClient) ResponseTypes() []oidc.ResponseType {
+	return c.responseTypes
+}
+
+func (c *ConfClient) GrantTypes() []oidc.GrantType {
+	return c.grantTypes
+}
+
+func (c *ConfClient) DevMode() bool {
+	return c.devMode && appConfig.Debug
```

Comment:
```suggestion
	return c.devMode
```

Incomprehensible code. 

Until I read this line, I wouldn't expect it to reflect the application settings. To make the intention clearer, set `appCofnig.Debug` to `devMode` only during initialization.